typescript/ccip-server/src/services/CallCommitmentsService.ts

```
@@ -39,15 +41,21 @@ const CommitmentRecordSchema = PostCallsSchema.extend({
+    private multiProvider: MultiProvider,
+    baseUrl: string,
+  ) {
     super();
-    this.registerRoutes(this.router);
+    this.registerRoutes(this.router, baseUrl);
   }
 
-  static async initialize() {
+  static async initialize(namespace: string) {
     const env = EnvSchema.parse(process.env);
     const multiProvider = await this.getMultiProvider(env.REGISTRY_URI);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)